bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/channelhelper/EventChannelHelper.java

```diff
@@ -36,8 +37,8 @@
  */
 @NonNullByDefault
 public class EventChannelHelper extends ChannelHelper {
-    private boolean isLocal;
-    private @Nullable String vpnUrl, localUrl;
+    private @NonNullByDefault({}) String vpnUrl;
```

Comment:
Please avoid bypassing the null checks. I would prefer to keep a `@Nullable` for `vpnUrl`.

Reply:
Done